02_configure_host.sh

```diff
@@ -275,33 +272,53 @@ sleep 5
 
 
 # Clone all needed repositories (CAPI, CAPM3, BMO, IPAM)
+# The repos cloned under M3PATH has two functions, building images and
+# providing manifest generation functionality
 mkdir -p "${M3PATH}"
-clone_repo "${BMOREPO}" "${BMOBRANCH}" "${BMOPATH}" "${BMOCOMMIT}"
+# When building local images make sure FORCE_REPO_UPDATE is set to 'false'
+# otherwise clone_repo will overwrite the content of whatever is at the end
+# of the path
```

Comment:
Same comment as in the other PR, suggesting to make this a function to avoid repetive code and repetive typos :)

Reply:
I have added the function.